hole/answers/rijndael-s-boxes.txt

```diff
@@ -0,0 +1,35 @@
+      0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f
```

Comment:
Would it be better to reduce the potential hole size by simplifying this to just a comma separated list of the hex bytes?

Reply:
What about
```
63 7c 77 ...
ca 82 c9 ...
...
```

Reply:
SGTM!